cpg_pipes/pipeline/cli_opts.py

```diff
@@ -0,0 +1,239 @@
+            '-S',
+            'skip_samples',
+            multiple=True,
+            help='Don\'t process specified samples. Can be set multiple times.',
+        ),
+        click.option(
+            '--only-sample',
+            '-s',
+            'only_samples',
+            multiple=True,
+            help='Only take these samples (can be set multiple times)',
+        ),
+        click.option(
+            '--force-sample',
+            'force_samples',
+            multiple=True,
+            help='Force reprocessing these samples. Can be set multiple times.',
+        ),
+        click.option(
+            '--version',
+            '--output-version',
+            'version',
+            type=str,
+            help='Pipeline version. Default is a timestamp',
+        ),
+        click.option(
+            '--storage-policy',
+            'storage_policy_type',
+            type=choice_from_enum(StoragePolicyType),
+            callback=val_to_enum(StoragePolicyType),
+            default=StoragePolicyType.CPG.value,
+            help='Storage policy is used to determine bucket names for intermediate '
+            'and output files',
+        ),
+        click.option(
+            '--cloud',
+            'cloud',
+            type=choice_from_enum(Cloud),
+            callback=val_to_enum(Cloud),
+            default=Cloud.GS.value,
+            help='Cloud storage provider',
+        ),
+        click.option(
+            '--status-reporter',
+            'status_reporter_type',
+            type=choice_from_enum(StatusReporterType),
+            callback=val_to_enum(StatusReporterType),
+            help='Use a status reporter implementation to report jobs statuses',
+        ),
+        click.option(
+            '--input-provider',
+            'input_provider_type',
+            type=choice_from_enum(InputProviderType),
+            callback=val_to_enum(InputProviderType),
+            default=InputProviderType.SMDB.value,
+            help=f'Source of inputs. '
+            f'For "--input-source={InputProviderType.CSV.value}", '
+            f'use --input-csv to specify a CSV file location',
+        ),
+        click.option(
+            '--input-csv',
+            'input_csv',
+            help=f'CSV file to provide with --input-provider={InputProviderType.CSV.value}',
+        ),
+        click.option(
+            '--keep-scratch/--remove-scratch',
+            'keep_scratch',
+            default=False,
+            is_flag=True,
+        ),
+        click.option('--dry-run', 'dry_run', is_flag=True),
+        click.option(
+            '--skip-samples-with-missing-input',
+            'skip_samples_with_missing_input',
+            default=False,
+            is_flag=True,
+            help='For the first (not-skipped) stage, if the input for a target does not'
+            'exist, just skip this target instead of failing. E.g. if the first'
+            'stage is Align, and sequence.meta files for a sample do not exist,'
```

Comment:
```suggestion
            help='For the first (not-skipped) stage, if the input for a target does not '
            'exist, just skip this target instead of failing. E.g. if the first '
            'stage is Align, and sequence.meta files for a sample do not exist, '
```